youbet/src/Bet.sol

```diff
@@ -63,215 +61,8 @@ contract Bet is Initializable, OwnableUpgradeable, UUPSUpgradeable {
         address newImplementation
     ) internal override onlyOwner {}
 
-    function createGoalSolo(
-        string memory _name,
-        string memory _description,
-        uint _requiredStake,
-        uint _taskCount
-    ) public {
-        _createGoal(
-            _name,
-            _description,
-            _requiredStake,
-            _taskCount,
-            GoalType.Solo
-        );
-    }
-
-    function createGoal(
-        string memory _name,
-        string memory _description,
-        uint _requiredStake,
-        uint _taskCount
-    ) public {
-        _createGoal(
-            _name,
-            _description,
-            _requiredStake,
-            _taskCount,
-            GoalType.Gambling
-        );
-    }
-
-    function _createGoal(
-        string memory _name,
-        string memory _description,
-        uint _requiredStake,
-        uint _taskCount,
-        GoalType _goalType
-    ) public {
-        uint goalId = goals.length;
-        Goal storage newGoal = goals.push();
-        newGoal.id = goalId;
-        newGoal.name = _name;
-        newGoal.description = _description;
-        newGoal.requiredStake = _requiredStake;
-        newGoal.creator = msg.sender;
-        newGoal.taskCount = _taskCount;
-        newGoal.goalType = _goalType;
-        newGoal.completed = false;
-
-        emit GoalCreated(
-            goalId,
-            _name,
-            _description,
-            _requiredStake,
-            _taskCount,
-            _goalType,
-            msg.sender
-        );
-    }
-
     event DebugLog(uint msgValue, uint requiredStake);
```

Comment:
This line is useless too?

Reply:
I don't understand where you are referring ?  All the code goal related will be useless, won't it ?

Reply:
I mean DebugLog should be deleted?